### PR TITLE
DEV-4509-make-subjects-pretty

### PR DESCRIPTION
**Pull request by nprimo**
I run the following script to reformat all the subjects inside `/public/subject/blockchain/`.
The objective is to have all the `README.md` `prettier` compliant and no `H1` block ("demoted" to `H2`)

```bash
#!/usr/bin/env bash 

IFS='
'

if [[ $# -lt 1 ]]; then
    echo "Run $0 with one or more aguments"
    exit 1
fi

format () {
    # assume prettier is installed 
    npx prettier -w "$1"
    # From H1 to H2 (only .md files)
    if [[ "$1" =~ .*\.md$ ]]; then
        sed -i 's:^# :## :' "$1"
    fi
}

for file in $@; do 
    if [[ !(-f $file) ]]; then 
       continue 
     fi
     format "$file"
done
```

### CODE REVIEW

File: subjects/blockchain/financial-instruments/README.md [hunk 1]
```diff
@@ -16,56 +16,56 @@ The network must have a minimum of 3 validating nodes. A script must facilitate
...
-* **Bonds: **A smart contract represents all the outstanding bonds from an issuer. Each bond has a unique serial number, a current principal, an interest rate, an issuance date, a maturity date, a current owner, and if it has been repaid. For simplicity, we assume each bond to be issued for one year, requiring only one payment. 
+- **Stablecoins**: It must be a standard fungible token (ERC20 or equivalent). It initially has 1000 units. The creator of the stablecoin can issue or remove units.
+- **Shares**: Shares are a fungible token for each company. Occasionally, its issuer can do a dividend payout. In that case, the issuer sends the stablecoin to the share contract, and each owner can retrieve proportionally to its possessions.
+- **Bonds: **A smart contract represents all the outstanding bonds from an issuer. Each bond has a unique serial number, a current principal, an interest rate, an issuance date, a maturity date, a current owner, and if it has been repaid. For simplicity, we assume each bond to be issued for one year, requiring only one payment.
 
 ## Populating
```
[the remark this answers is not in the document]
Comment (davhojt) [reply subjects/blockchain/financial-instruments/README.md:23]: There's a few instances of it throughout the blockchain content. They'll likely need to be h3.

**Review by davhojt — CHANGES_REQUESTED** [review]
Looks okay, with the exception of h2 tags. They don't render. Probably need to be h3.

File: subjects/blockchain/financial-instruments/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nprimo) [reply subjects/blockchain/financial-instruments/README.md]: I have updated all the instances of H2 to H3 (besides the main title, according to the rest of our curriculum subjects) - thank you for having caught this issue :)

File: subjects/blockchain/financial-instruments/README.md [hunk 2]
```diff
@@ -131,52 +131,48 @@ Total available</p>
...
-* A table with the assets usable on the platform, their type, the address of their smart contract, and their price history.
-* A table of the users of the platform registered after the first connection. They must provide their legal name and upload a “passport” picture. No check is made when login in, but the picture is stored for future reference
-* A table of all ongoing sell and buy offers on the platform
 
+- A table with the assets usable on the platform, their type, the address of their smart contract, and their price history.
+- A table of the users of the platform registered after the first connection. They must provide their legal name and upload a “passport” picture. No check is made when login in, but the picture is stored for future reference
+- A table of all ongoing sell and buy offers on the platform
 
-## The vault smart contract
+#### The vault smart contract
```
Comment (MSilva95) [inline subjects/blockchain/financial-instruments/README.md:152]: Does it need to be 4 hashes or 3 like the others?

File: subjects/blockchain/financial-instruments/README.md [hunk 3]
```diff
@@ -131,52 +131,48 @@ Total available</p>
...
 
-The vault smart contract will receive the various financial assets. In a simple form, it includes an `operateWithdrawal(user, asset, amount)` reserved to the platform issuer allows the platform to send assets to the user, after verifying that the funds were theirs and that there are no pending orders. 
+The vault smart contract will receive the various financial assets. In a simple form, it includes an `operateWithdrawal(user, asset, amount)` reserved to the platform issuer allows the platform to send assets to the user, after verifying that the funds were theirs and that there are no pending orders.
 
 _Optionally, a more sophisticated security model can be proposed._
 
+#### Trade execution
```
Comment (MSilva95) [inline subjects/blockchain/financial-instruments/README.md:158]: same here

File: subjects/blockchain/node-dashboard/README.md [hunk 4]
```diff
@@ -1,43 +1,42 @@
...
+- Their latest block
+- Their list of connected peers
+- The number of transactions in their mempool
+- Their current CPU and memory consumption (Hint: the outputs of commands such as `docker stats` or `ps` can be used)
+- Their corresponding Ethereum address and its balance.
 
+### Scenario
 
-## Scenario
+To analyze our network, we need to simulate some transactions. Benchy should propose the following scenarios available with the command `benchy scenario X`. Each scenario must provide feedback on its execution, including updated balances after transfers. 0. Initialise the network by letting it run for a few minutes. Validating nodes must have Eth available as reward or part of the initial configuration.
```
Comment (MSilva95) [inline subjects/blockchain/node-dashboard/README.md:27]: here i think the "0. Initialise the network by letting it run for a few minutes. Validating nodes must have Eth available as reward or part of the initial configuration."  needs to be in the next line like the others, and maybe substitute the number with normal  -

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Overall it is good, just left some inline comments

File: subjects/blockchain/financial-instruments/README.md (hunk 2 above)
Comment (nprimo) [reply subjects/blockchain/financial-instruments/README.md:152]: done

File: subjects/blockchain/financial-instruments/README.md (hunk 3 above)
Comment (nprimo) [reply subjects/blockchain/financial-instruments/README.md:158]: I moved it back to h3

File: subjects/blockchain/node-dashboard/README.md (hunk 4 above)
Comment (nprimo) [reply subjects/blockchain/node-dashboard/README.md:27]: I implemented everything you suggested! :)

**Review by MSilva95 — APPROVED** [review]
LGTM!

**Pull request merged**